resources/js/pages/Welcome.vue

```diff
@@ -1,6 +1,10 @@
 <template>
   <v-container>
-    Hello world!
+    <v-card evaluation="4">
```

Comment:
the property name is `elevation` not ~evaluation~ and please make it 5 (same as `Admin/Dashboard`)
https://vuetifyjs.com/en/components/cards/#usage